api/v1/src/accounts/index.js

```diff
@@ -682,11 +742,260 @@ accounts.get(['/projects/:projectId/accounts:register', '/accounts:register'], a
     }
 });
 
-// Backwards compatibility for marketplace
+/**
+ * @swagger
+ *
+ * /accounts:register:
+ *   options:
+ *     summary: CORS support
+ *     description: Enable CORS by returning correct headers
+ *     operationId: optionsRegisterAccount
+ *     security: [] # no security for preflight requests
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: Default response for CORS method
+ *         headers:
+ *           Access-Control-Allow-Headers:
+ *             type: "string"
+ *           Access-Control-Allow-Methods:
+ *             type: "string"
+ *           Access-Control-Allow-Origin:
+ *             type: "string"
+ *   post:
+ *     summary: Register a marketplace account based off request body
+ *     description: Returns a redirect response
+ *     operationId: registerAccount
+ *     security: [] # no security for preflight requests
```

Comment:
This POST is invoked by a non-logged in user therefore no security should be enabled here.